recipes/enzy_htp/license.md

```diff
@@ -0,0 +1,73 @@
+Please Note: This “Non-Exclusive Non-Commercial Academic Software License Agreement” is available only to non-profit academic and/or research institutions who use the Software solely for internal non-commercial research purposes.  In the event the Licensee wishes to obtain a commercial use license, the Licensee must contact Vanderbilt's Center for Technology Transfer and Commercialization, 1207 17th Avenue South, Suite 105, Nashville, Tennessee 37212 (cttc@vanderbilt.edu). 
+
+
+
+EnzyHTP
+=================================================================
+Non-Exclusive Non-Commercial Academic Software License Agreement
+=================================================================
+This License Agreement ("Agreement") is entered into between Vanderbilt University, a private, non-profit institution of higher education in Nashville, Tennessee with offices at 1207 17th Avenue South, Suite 105, Nashville, Tennessee 37212 ("Vanderbilt") and the person representing the non-profit research and/or academic institution that registers to use the Software ("Licensee"). Prior to such registration in accordance with this Agreement, the Licensee shall have no rights or license in connection with the Software or this Agreement.
+
```

Comment:
I don't think that we can accept this license because members of conda-forge will not and probably cannot enter a license agreement with anyone.

Please wait until the license change has been completed and update it then.